### ISSUE TITLE
Followers stop following and stay in loby

### ISSUE THREAD

**Issue by Kojwishing** [kolton/d2bot-with-kolbot#2361]
So i run a lead with 3 followers. all 3 followers are ran from botchannel. for the first few games they will all join leads game and run as intended with no issues. after a while of running, 5+ runs, all the followers stop joining the game and just sit idle in the lobby, i also have a friend who follows on 2 followers, and his followers also will idle in the lobby. im not sure of this issued since ive never ran into this problem. and it just started happening after all the recent realm downs within the Covid-19.

**Comment by Kojwishing** [comment]
Is this github a deserted site? i mean. been posted for 4 days and no reply? maybe just can this whole bot idea? good lord.

**Comment by Kojwishing** [comment]
Still have this problem, i've been going over everything i can think of. its not only my program doing this as well. i have a friend following me on his program states away, and his 2 followers also just stop joining games and idling in lobby spamming /f l. ill more than likely drop botting all together with all the issues ive had recently with no outsource help.

**Comment by WiZzMaN47** [comment]
Have you set config.leader in char config file?

**Comment by Kojwishing** [comment]
yeah, i mean its weird. its been running fine the past 2 years. but now all the sudden i go to classic and it changed everything. i havnt tested expac yet to see if its doing it back over there, but on classic its doing it all the time and i never had a problem on either expac or classic. and never had to change any configs ;(
